listenbrainz/webserver/static/js/src/recommendations/Recommendations.tsx

```diff
@@ -0,0 +1,320 @@
+/* eslint-disable jsx-a11y/anchor-is-valid,camelcase */
+
+import { AlertList } from "react-bs-notifier";
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import * as _ from "lodash";
+import BrainzPlayer from "../BrainzPlayer";
+import APIService from "../APIService";
+import Loader from "../components/Loader";
+import RecommendationCard from "./RecommendationCard";
+
+export interface RecommendationsProps {
+  apiUrl: string;
+  recommendations?: Array<Recommendation>;
+  profileUrl?: string;
+  spotify: SpotifyUser;
+  user: ListenBrainzUser;
+  webSocketsServerUrl: string;
+  currentUser?: ListenBrainzUser;
+}
+
+export interface RecommendationsState {
+  alerts: Array<Alert>;
+  currentRecommendation?: Recommendation;
+  direction: BrainzPlayDirection;
+  recommendations: Array<Recommendation>;
+  loading: boolean;
+  currRecPage?: number;
+  totalRecPages: number;
+}
+
+export default class Recommendations extends React.Component<
+  RecommendationsProps,
+  RecommendationsState
+> {
+  private brainzPlayer = React.createRef<BrainzPlayer>();
+  private recommendationsTable = React.createRef<HTMLTableElement>();
+  private APIService: APIService;
+
+  private socket!: SocketIOClient.Socket;
```

Comment:
Do we need sockets on recs page?

Reply:
I don't exactly understand the purpose, so I just included them

Reply:
Sockets are used for real time updates etc. We wont need it here